src/tests/stress/pedsim/Ped_util/Regions.enc

```diff
@@ -0,0 +1,480 @@
+module Regions
+import Agent_passive
+import Global_funs
+import Quad_tree
+
+fun regions(var agents : [Agent], boxes : int) : unit
+  var box = new Tiling_box(agents, boxes)
+  box!move(10000)
```

Comment:
That's part of the problem spec, 10000 time steps. I can lift it into a the test start if you like.